app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -224,9 +224,9 @@ class PinPasswordActivityTest {
       closeSoftKeyboard()
       onView(withId(R.id.pin_password_input_pin_edit_text)).perform(closeSoftKeyboard())
         .perform(editTextInputAction.appendText("54321"), closeSoftKeyboard())
-      onView(withText(context.getString(R.string.pin_password_incorrect_pin))).check(
```

Comment:
Out of curiosity, does this old check work if put back? We should be using the text input layout version, but I'm curious.

Also, have you checked via step debugging whether the error actually is set by the time it's checking for it to be present?

Finally, have you verified the behavior on both Robolectric and Espresso? Is it failing in the same way on both?

Reply:
> Out of curiosity, does this old check work if put back? We should be using the text input layout version, but I'm curious.
> 
> Also, have you checked via step debugging whether the error actually is set by the time it's checking for it to be present?
> 
> Finally, have you verified the behavior on both Robolectric and Espresso? Is it failing in the same way on both?

Tests are passing in espresso and failing in robolectric. I have tried to check whether view as error text or not by asserting the TextInputLayout and in that it TextInputLayout.error is returning null.